script.js

```diff
@@ -1,4 +1,77 @@
+//user inputs their name
+var startGame = `please enter your name`;
+var winScore = 0;
+var playerTries = 0;
+// user inputs scissors, paper or stone, output will determine win lose or draw or wrong spelling
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = ``;
+  playerTries = playerTries + 1;
+  //do this if game mode is waiting for enter your name
+  if (startGame == `please enter your name`) {
+    //set the name
+    userName = input;
+    //now that we have the name, switch mode
+    startGame = `SPS game`;
+    playerTries = playerTries - 1;
+    myOutputValue = `Welcome  ` + userName;
+  }
+  //if gamemode is sps game
+  else if (startGame == `SPS game`) {
+  }
+  var computerChoice = SPS();
+  var myOutputValue =
+    `Hello ` +
+    userName +
+    `    please enter any of the following: scissors, paper, stone to proceed!`;
+  //scissors =1 SPS
+  //paper = 2
+  //stone =3
```

Comment:
you can implement this logic in the SPS function. Using if statement.
if(randomIntegar==1){
return 'scissors'
}else if (randomIntegar==2){
return 'paper'
}else {
return 'stone'
}